### PR TITLE
Moving doc from wiki to repo

### PR DESCRIPTION
**Pull request by fgalan**
The PR (which corrresponds to an structured dump of the current doc snapshot at FIWARE wiki to Markdown) is too large (more than 7,000 lines ;) to do an in-deep review. Review should focus in structural aspects, checking if the layout of the documentation conforms with the new way of documenting defined in FIWARE.

Once this PR lands in develop, more PRs will come to polish the document, e.g. fix intra-documentation broken links, make code fragments more tidy, remove lefovers, etc.

After LGTM this PR:

* Will replace README.md by README_new.md
* Will be squashed to only one commit.

### CODE REVIEW

File: README_new.md [hunk 1]
```diff
@@ -0,0 +1,617 @@
...
+
+The unit_test makefile target is used for running the unit tests:
+
+    make functional_test INSTALL_DIR=~
+
+Please have a look to the section [on building the source code](#from-sources) in order to get more information about how to prepare the environment to run the unit_test target.
+
+## Advanced topics:
+
+* Installation and aministration
```
Comment (fgalan) [inline README_new.md]: Topics have been ordered in relevance order (most relevant: up in the list) within each category (admin and API). Any feedback about this priority is welcome.

File: README_new.md [hunk 2]
```diff
@@ -0,0 +1,617 @@
...
+
+    Checking contextBroker...                         Running
+
+If broker is not running you will get:
+
+    Checking contextBroker...                         Service not running
+
+## API Overview
+
+In order to create an entity (Room1) with two attributes (temperature and pressure):
```
Comment (fgalan) [inline README_new.md]: In is hard to which level cover the API in an Overview section... I think that these tree operations (create, query and updated) are the basic ones, but if you think some additional one should be included without making the section unnecesary complex, let me know. In addition, note that a link to the Quick Start guide (that shows how to use these operations with a real Orion instance) is included.

File: README_new.md [hunk 3]
```diff
@@ -0,0 +1,617 @@
+# Orion Context Broker
```
Comment (jmcanterafonseca) [inline README_new.md]: why are we renaming README to README_new?

File: README_new.md (hunk 2 above)
Comment (jmcanterafonseca) [inline README_new.md]: Can we just put here a link to the Apiary Cookbook?

File: README_new.md (hunk 1 above)
Comment (jmcanterafonseca) [inline README_new.md]: nit: administration

File: README_new.md (hunk 3 above)
Comment (fgalan) [inline README_new.md]: To make easir the review, avoding a very noisy diff in the PR. But the idea is to use only README.md at the end, note from PR body:

> After LGTM this PR:
> *    Will replace README.md by README_new.md
> *    Will be squashed to only one commit.

File: README_new.md (hunk 2 above)
Comment (fgalan) [inline README_new.md]: I understand that the cookbook is part of the API Walkthorugh/Examples that, as far as I understand by https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Developer_Guidelines, is in a different section (just below this one).

File: README_new.md (hunk 1 above)
Comment (fgalan) [inline README_new.md]: Fixed in https://github.com/telefonicaid/fiware-orion/commit/2c7fe528bd534d0d743e67b63752337f8a87026d

File: README_new.md [hunk 4]
```diff
@@ -0,0 +1,617 @@
...
+
+This is the code repository for the Orion Context Broker, the reference implementation of the Publish/Subscribe Context Broker GE.
+
+This project is part of FIWARE.
```
Comment (jmcanterafonseca) [inline README_new.md]: maybe you can create an hyperlink with the FIWARE word and save the line below

File: README_new.md [hunk 5]
```diff
@@ -0,0 +1,617 @@
...
+
+* [FIWARE Site](http://www.fiware.org)
+* [FIWARE Catalogue for Orion](http://catalogue.fiware.org/enablers/publishsubscribe-context-broker-orion-context-broker)
+
+Any feedback on this documentation is highly welcome, including bugs, typos
+or things you think should be included but aren't. You can use [github issues](https://github.com/telefonicaid/fiware-orion/issues/new) to provide feedback.
+
+For documentation previous to Orion 0.23.0 please check the old manuals at FIWARE public wiki:
```
Comment (jmcanterafonseca) [inline README_new.md]: maybe you can avoid the word 'old' as it may have negative connotations

File: README_new.md [hunk 6]
```diff
@@ -0,0 +1,617 @@
...
+
+* [Orion Context Broker - Installation and Administration Guide](https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Publish/Subscribe_Broker_-_Orion_Context_Broker_-_Installation_and_Administration_Guide)
+* [Orion Context Broker - User and Programmers Guide](https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Publish/Subscribe_Broker_-_Orion_Context_Broker_-_User_and_Programmers_Guide)
+
+## GEi overall description
+
+Orion is a C++ implementation of the NGSI9/10 REST API binding developed as a part of the FIWARE platform.
+
+The Orion Context Broker is an NGSI9/10 server implementation to manage context information and context information availability. Using the Orion Context Broker, you are able to register context elements and manage them through updates and queries. In addition, you can subscribe to context information so when some condition occurs (e.g. an interval of time has passed or the context elements have changed) you receive a notification. These usage scenarios and the Orion Context Broker features are described in this document.
```
Comment (jmcanterafonseca) [inline README_new.md]: to manage context information and its availability

File: README_new.md [hunk 7]
```diff
@@ -0,0 +1,617 @@
...
+
+If this is your first contact with Orion Context Broker, it is highly recommended to have a look to the brief [Quick Start guide](doc/md/quick_start_guide.md).
```
Comment (jmcanterafonseca) [inline README_new.md]: your first approach to ...

File: README_new.md (hunk 4 above)
Comment (fgalan) [inline README_new.md]: Fixed in d356452.

File: README_new.md (hunk 5 above)
Comment (fgalan) [inline README_new.md]: Fixed in de740476a6006285a458cff382e091e1c9c191f5 and 217e8111c2e2b1ca24380ba30e4dfd75f3109301

File: README_new.md (hunk 6 above)
Comment (fgalan) [inline README_new.md]: Fixed in de740476a6006285a458cff382e091e1c9c191f5 and 217e8111c2e2b1ca24380ba30e4dfd75f3109301

File: README_new.md (hunk 7 above)
Comment (fgalan) [inline README_new.md]: Fixed in de740476a6006285a458cff382e091e1c9c191f5 and 217e8111c2e2b1ca24380ba30e4dfd75f3109301

File: doc/md/admin/upgrading_crossing_0-14-1.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
+# Upgrading to 0.14.1 and beyond from any pre-0.14.1 version
```
Comment (jmcanterafonseca) [inline doc/md/admin/upgrading_crossing_0-14-1.md]: I don't know if it is worth keeping this documentation from such old versions. but you know better than me the history around the component

File: doc/md/user/cprs.md [hunk 9]
```diff
@@ -0,0 +1,188 @@
+# Context Providers registration and request forwarding
```
Comment (jmcanterafonseca) [inline doc/md/user/cprs.md]: maybe we can name this file as context_providers.md, clearer isn't it?

Comment (fgalan) [inline doc/md/user/cprs.md]: Ok.

File: doc/md/admin/upgrading_crossing_0-14-1.md (hunk 8 above)
Comment (fgalan) [inline doc/md/admin/upgrading_crossing_0-14-1.md]: There are people still with old versions (e.g. some Xifi nodes at least are using Orion 0.13.0) so better to have this documentation at hand as long as the target (new) version in migrations is the one covered by the markdown documentation.

NTC

File: doc/md/user/http_and_ngsi_sc.md [hunk 10]
```diff
@@ -0,0 +1,80 @@
+# HTTP and NGSI response codes
```
Comment (jmcanterafonseca) [inline doc/md/user/http_and_ngsi_sc.md]: sc what does it stand for?

**Comment by jmcanterafonseca** [comment]
Fermin, very good work. Modulo the comments reported I think we can land this and continue working. As I said offline we need to work on a refactored README and perhaps we need to refactor the doc folder as well to something like

doc/spec
doc/manuals

LGTM

File: doc/md/user/http_and_ngsi_sc.md (hunk 10 above)
Comment (fgalan) [inline doc/md/user/http_and_ngsi_sc.md]: sc = status code

NTC

File: doc/md/user/cprs.md (hunk 9 above)
Comment (fgalan) [inline doc/md/user/cprs.md]: Fixed in 3c587d4

File: README_new.md (hunk 2 above)
Comment (fgalan) [inline README_new.md]: NTC

File: README_new.md (hunk 3 above)
Comment (fgalan) [inline README_new.md]: Renaming done at the end.

File: README_new.md (hunk 7 above)
Comment (kzangeli) [inline README_new.md]: On 25/06/15 16:56, Jose M. Cantera wrote:

In README_new.md<https://github.com/telefonicaid/fiware-orion/pull/1024#discussion_r33263699>:

> +
> +Any feedback on this documentation is highly welcome, including bugs, typos
> +or things you think should be included but aren't. You can use [github issues](https://github.com/telefonicaid/fiware-orion/issues/new) to provide feedback.
> +
> +For documentation previous to Orion 0.23.0 please check the old manuals at FIWARE public wiki:
> +
> +* [Orion Context Broker - Installation and Administration Guide](https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Publish/Subscribe_Broker_-_Orion_Context_Broker_-_Installation_and_Administration_Guide)
> +* [Orion Context Broker - User and Programmers Guide](https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Publish/Subscribe_Broker_-_Orion_Context_Broker_-_User_and_Programmers_Guide)
> +
> +## GEi overall description
> +
> +Orion is a C++ implementation of the NGSI9/10 REST API binding developed as a part of the FIWARE platform.
> +
> +The Orion Context Broker is an NGSI9/10 server implementation to manage context information and context information availability. Using the Orion Context Broker, you are able to register context elements and manage them through updates and queries. In addition, you can subscribe to context information so when some condition occurs (e.g. an interval of time has passed or the context elements have changed) you receive a notification. These usage scenarios and the Orion Context Broker features are described in this document.
> +
> +If this is your first contact with Orion Context Broker, it is highly recommended to have a look to the brief [Quick Start guide](doc/md/quick_start_guide.md).


your first approach to ...

—
Reply to this email directly or view it on GitHub<https://github.com/telefonicaid/fiware-orion/pull/1024/files#r33263699>.


Better with "your first contact with".

"Approach" as a noun means 'a way to do things', and this is not what we want here ...



________________________________

Este mensaje y sus adjuntos se dirigen exclusivamente a su destinatario, puede contener información privilegiada o confidencial y es para uso exclusivo de la persona o entidad de destino. Si no es usted. el destinatario indicado, queda notificado de que la lectura, utilización, divulgación y/o copia sin autorización puede estar prohibida en virtud de la legislación vigente. Si ha recibido este mensaje por error, le rogamos que nos lo comunique inmediatamente por esta misma vía y proceda a su destrucción.

The information contained in this transmission is privileged and confidential information intended only for the use of the individual or entity named above. If the reader of this message is not the intended recipient, you are hereby notified that any dissemination, distribution or copying of this communication is strictly prohibited. If you have received this transmission in error, do not read it. Please immediately reply to the sender that you have received this communication in error and then delete it.

Esta mensagem e seus anexos se dirigem exclusivamente ao seu destinatário, pode conter informação privilegiada ou confidencial e é para uso exclusivo da pessoa ou entidade de destino. Se não é vossa senhoria o destinatário indicado, fica notificado de que a leitura, utilização, divulgação e/ou cópia sem autorização pode estar proibida em virtude da legislação vigente. Se recebeu esta mensagem por erro, rogamos-lhe que nos o comunique imediatamente por esta mesma via e proceda a sua destruição

**Pull request merged**